src/components/RemoveButton/index.jsx

```diff
@@ -17,7 +17,7 @@ const RemoveButton = ({
     >
       <Icon
         id="remove-field"
-        className={['fa', 'fa-close', 'fa-2x', 'text-danger']}
+        className={['fa', 'fa-close']}
```

Comment:
```suggestion
        className={['fa', 'fa-close', 'fa-5x', 'text-danger', 'bg-success']}
```


<img width="649" alt="Screen Shot 2019-04-11 at 9 42 07 AM" src="https://user-images.githubusercontent.com/14864970/55962032-20d3f400-5c3e-11e9-90df-67990118bbed.png">
I think this is what we want